### PR TITLE
DHFPROD-9253: Testing sample data in a matching step it deletes more URIs than the one I select

### PR DESCRIPTION
**Pull request by senumulapally**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

**Review by littlecherro — COMMENTED** [review]
Could you check jenkins? The test you changed is failing :( also in my pc

File: marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/matching.spec.tsx [hunk 1]
```diff
@@ -26,8 +26,9 @@ const allDataMatchedResults = [{ruleset: "lname - Exact", matchType: "Exact 0",
 
 const urisMerged = ["/json/persons/first-name-double-metaphone1.json", "/json/persons/first-name-double-metaphone2.json"];
 const uris = ["/json/persons/first-name-double-metaphone1.json", "/json/persons/first-name-double-metaphone2.json", "/json/persons/last-name-plus-zip-boost1.json", "/json/persons/last-name-plus-zip-boost2.json", "/json/persons/last-name-dob-custom1.json", "/json/persons/last-name-dob-custom2.json", "/json/persons/first-name-synonym1.json", "/json/persons/first-name-synonym2.json"];
-const compareValuesData = [{propertyName: "id", uriValue1: "empty", uriValue2: "empty"}, {propertyName: "fname", uriValue1: "Alexandra", uriValue2: "Alexandria"}, // eslint-disable-line @typescript-eslint/no-unused-vars
-  {propertyName: "lname", uriValue1: "Wilson", uriValue2: "Wilson"}, {propertyName: "Address", uriValue1: "123 Wilson St", uriValue2: "123 Wilson Rd"}];
+const urisDummy = ["dummy1", "dummy2"];
+const compareValuesData = [{propertyName: "id", uriValue1: "empty", uriValue2: "empty"}, {propertyName: "fname", uriValue1: "Alexandria", uriValue2: "Alexandria"}, // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (briantang) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/matching.spec.tsx:30]: Something you changed to the variable here is the reason for the test failing. I think a conflict was resolved wrong. Once you rollback the compareValuesData the test should pass.

Comment (senumulapally) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/matching.spec.tsx]: Resolved the merge conflict.

**Review by briantang — APPROVED** [review]
Looks good. Remaining failures unrelated.

**Review by littlecherro — APPROVED** [review]

**Pull request merged**